### DISCUSSION TITLE
Does the length of abstracts matter?

### DISCUSSION THREAD

**Discussion by SagevdBrand** [asreview/asreview#572] [Q&A] [answered]
Sometimes I have abstracts that are only 150 words, or they are quite long (300 words), does this have an effect on how relevant an abstract is regarded? Would the longer abstract gain more 'weight' so to speak?

**Comment by bagheria** [comment, marked as answer]
We consider text data (abstracts) high-dimensional in the AI-based ASReview tool, as each word in the abstract can be one dimension in your data set. On the other hand abstracts are sparse, meaning that they only have a small portion of the entire vocabulary of your data set. That said, the more informative words in an abstract the better representation for the machine learning methods in ASReview, and hence a more reliable decision regarding their relevancy.
